lms/djangoapps/branding/tests/test_page.py

```diff
@@ -163,82 +163,86 @@ def test_course_with_prereq(self):
-    def test_course_cards_sorted_by_default_sorting(self):
-        response = self.client.get('/')
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'index.html')
-
-        # by default the courses will be sorted by their creation dates, earliest first.
-        self.assertEqual(context['courses'][0].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][1].id, self.starting_later.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-        # check the /courses view
-        response = self.client.get(reverse('branding.views.courses'))
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'courseware/courses.html')
-
-        # by default the courses will be sorted by their creation dates, earliest first.
-        self.assertEqual(context['courses'][0].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][1].id, self.starting_later.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-    @patch('student.views.render_to_response', RENDER_MOCK)
-    @patch('courseware.views.render_to_response', RENDER_MOCK)
-    @patch.dict('django.conf.settings.FEATURES', {'ENABLE_COURSE_SORTING_BY_START_DATE': False})
-    def test_course_cards_sorted_by_start_date_disabled(self):
-        response = self.client.get('/')
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'index.html')
-
-        # now the courses will be sorted by their announcement dates.
-        self.assertEqual(context['courses'][0].id, self.starting_later.id)
-        self.assertEqual(context['courses'][1].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-        # check the /courses view as well
-        response = self.client.get(reverse('branding.views.courses'))
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'courseware/courses.html')
-
-        # now the courses will be sorted by their announcement dates.
-        self.assertEqual(context['courses'][0].id, self.starting_later.id)
-        self.assertEqual(context['courses'][1].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
+# THESE TESTS HAVE BEEN COMMENTED OUT BECAUSE FIND COURSES PAGE
+# HAS BEEN SWITCHED OVER TO JS AND EDX SEARCH.
+# AS OF NOW EDX SEARCH DOESN'T SUPPORT RESULTS ORDERING.
```

Comment:
I wouldn't phrase it like that - elastic search does have ordering based upon the scores of its results. Also, I think that these tests should remain in place in order to test when courseware discovery is OFF, that the behaviour is as prescribed before.

I would try overriding the settings to use the old list for these tests. If that doesn't work, I would mark the tests as @skip instead of commenting them out... having them as a large comment is far less useful than having them as skipped for a reason.

Basically, if we want to get rid of them, we should get rid of them... but I think we'd like to keep them, so have them running within the original context seems like the right thing to do.